src/makeCalculator.js

```diff
@@ -37,7 +37,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(action, number) {
+      this.result = action(this.result, number);
+
+      return this;
+    },
+
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => (b > 0 ? a / b : 'Cann`t be devided by 0'),
```

Comment:
The division by subzero numbers should be possible. Also, consider returning 0 instead of the string or throwing an error